src/components/InfinityModal/InfinityModalForm.tsx

```diff
@@ -0,0 +1,330 @@
+			<Field
+				name="untrainedTest"
+				id="untrainedTest"
+				label={untrainedTestLabel}
+				component={RenderCheckbox}
+				disabled={focus && Number(focus) > 0}
+			/>
+			<div className="fortune">
+				<h5 className="fortune-title">Fortune <InfoTooltip content={tooltip.fortuneInfo} /></h5>
+				<div className="infinity-radio-fields">
+					<label className="dice-row-label">
+						<Field
+							name="fortune"
+							component="input"
+							type="radio"
+							value="0"
+						/>
+							None
+						</label>
+					<label className="dice-row-label">
+						<Field
+							name="fortune"
+							component="input"
+							type="radio"
+							value="1"
+							onChange={() => Number(dice) < 3 ? change('dice', '3') : null}
+						/>
+							One
+						</label>
+					<label className="dice-row-label">
+						<Field
+							name="fortune"
+							component="input"
+							type="radio"
+							value="2"
+							onChange={() => Number(dice) < 4 ? change('dice', '4') : null}
+						/>
+							Two
+						</label>
+					<label className="dice-row-label">
+						<Field
+							name="fortune"
+							component="input"
+							type="radio"
+							value="3"
+							onChange={() => Number(dice) < 5 ? change('dice', '5') : null}
+						/>
+							Three
+						</label>
+				</div>
+			</div>
+			<div className="assistance">
+				<Accordion defaultActiveKey="0">
+					<Accordion.Toggle className="assistance-title--container" eventKey="1">
+						<h5 className="assistance-title">Assistance <InfoTooltip content={tooltip.assistanceInfo} /></h5>
+					</Accordion.Toggle>
+					<Accordion.Collapse eventKey="1">
+						<>
+							<div className="flex-center">
+								<div className="infinity-radio-fields infinity-radio-fields--dice-to-roll infinity-radio-fields--assistance">
+									<InfinityDiceRow
+										dice={assistanceDice}
+										diceMax={4}
+										fortune={0}
+										hoverState={assistanceHover}
+										handleOnHover={setAssistanceHover}
+										handleOnClick={handleAssistanceDiceChange}
+										isAssistance={true}
+									/>
+								</div>
+							</div>
+							<div className="skill-level-field infinity-skill-level-field infinity-skill-level-field--assistance">
+								<div className="infinity-field">
+									<Field
+										id="assistanceFocus"
+										name="assistanceFocus"
+										label={assistanceFocusLabel}
+										placeholder={focus}
+										component={renderInput}
+									/>
+								</div>
+								<div className="infinity-field">
+									<Field
+										id="assistanceTn"
+										name="assistanceTn"
+										label={assistanceTnLabel}
+										placeholder={tn}
+										component={renderInput}
+									/>
+								</div>
+							</div>
+						</>
+					</Accordion.Collapse>
+				</Accordion>
+			</div>
+		</Form>
+	);
+}
+
+interface ErrorPropTypes {
+	tn?: string;
+	focus?: string;
+	assistanceFocus?: string;
+	assistanceTn?: string;
+}
+
+const validate = (values: any) => {
```

Comment:
validate looks very similar like in Conan 2d20. I think we could use it as a external `util` function and import into the two forms.

Reply:
I agree.  This was all wholesale pulled from Conan section, so it makes sense there are areas of reusability